Frontend/bugfixes/iss300

Rintel opened the pull request.

mharasic committed:
removed ";" extra for Victor

mharasic committed:
removed ";" extra for Victor

Phil-Schmidt committed:
Hasty Hotfix for edit projects

-The site no longer crashes when the edit project-button is pressed
-It will no longer show the user, that the project does not exist; the loading spinner will spin for eternity.

Phil-Schmidt committed:
Hasty Hotfix for edit projects

-The site no longer crashes when the edit project-button is pressed
-It will no longer show the user, that the project does not exist; the loading spinner will spin for eternity.

mharasic committed:
minor bugfix and ";" removed

mharasic committed:
minor bugfix and ";" removed

Phil-Schmidt committed:
Merge pull request #308 from Drakulix/frontend/hotfix/editpage

Hasty Hotfix for edit projects

mharasic committed:
Moar (LESS) ";"

mharasic committed:
Moar (LESS) ";"

mharasic committed:
Minor bug fix

mharasic committed:
Minor bug fix

mharasic committed:
Merge branch 'frontend/bugfixes/iss300' of https://github.com/Drakulix/knex into frontend/bugfixes/iss300

Conflicts:
	web/src/components/common/Filters.jsx
	web/src/components/views/CreateProjectByPattern.jsx

Rintel committed:
Merge branch 'master' into frontend/bugfixes/iss300

mharasic committed:
Fixed minor BUG

mharasic committed:
Fixed minor BUG

mharasic committed:
Merge branch 'frontend/bugfixes/iss300' of https://github.com/Drakulix/knex into frontend/bugfixes/iss300

mharasic committed:
fff

mharasic committed:
fff

gitmirgut committed:
revert to old state from CreateProjectPattern.

gitmirgut committed:
revert to old state from CreateProjectPattern.

gitmirgut committed:
Merge branch 'master' into frontend/bugfixes/iss300

mharasic committed:
Fixed create project

mharasic committed:
Fixed create project

mharasic committed:
Merge branch 'frontend/bugfixes/iss300' of https://github.com/Drakulix/knex into frontend/bugfixes/iss300

Conflicts:
	web/src/components/views/CreateProjectByPattern.jsx

mharasic committed:
Minor bugfix

mharasic committed:
Minor bugfix

markwitz reviewed (dismissed):
rebase plz to not drop backend code

mharasic committed:
MOAR ; Remove

mharasic committed:
MOAR ; Remove

mharasic committed:
minor

mharasic committed:
minor

mharasic committed:
LESESS COLON

mharasic committed:
LESESS COLON

mharasic committed:
Capitalisation adopted

mharasic committed:
Capitalisation adopted

mharasic committed:
minor

mharasic committed:
minor

mharasic committed:
Added TagDiscovery

mharasic committed:
Added TagDiscovery

mharasic committed:
File Rename

mharasic committed:
File Rename

mharasic committed:
Refactoring

mharasic committed:
Refactoring

PSDavidSt committed:
added delete button

- added to any filter type
- changed display none to visibility hidden, so that delete button dont "moves"  arround if added or deleted

PSDavidSt committed:
added delete button

- added to any filter type
- changed display none to visibility hidden, so that delete button dont "moves"  arround if added or deleted

mharasic committed:
Performance Improve at ProjectContainer View

mharasic committed:
Performance Improve at ProjectContainer View

mharasic committed:
Half Fix edit project

mharasic committed:
Half Fix edit project

mharasic committed:
Merge branch 'frontend/bugfixes/iss300' of https://github.com/Drakulix/knex into frontend/bugfixes/iss300

mharasic committed:
Upload Error Narrowed

mharasic committed:
Upload Error Narrowed

mharasic committed:
Profile Refactoring

mharasic committed:
Profile Refactoring

mharasic committed:
Styled Admin view

mharasic committed:
Styled Admin view

mharasic committed:
Warnings and Formating

mharasic committed:
Warnings and Formating

mharasic committed:
AWESOME Line indent

mharasic committed:
AWESOME Line indent

mharasic committed:
removed nearly all ;

mharasic committed:
removed nearly all ;

mharasic committed:
Notifications fixes

mharasic committed:
Notifications fixes

mharasic committed:
removed Font Awesome

mharasic committed:
removed Font Awesome

mharasic committed:
cleaned dependencies

mharasic committed:
cleaned dependencies

mharasic committed:
Removed blinking of table due undefined

mharasic committed:
Removed blinking of table due undefined

mharasic committed:
removed idea

mharasic committed:
removed idea

mharasic committed:
DIRTY REBASE

mharasic committed:
DIRTY REBASE

mharasic commented:
FINISHED

mharasic committed:
REABSE

henderkes commented on tests/api/conftest.py:
```diff
@@ -90,10 +90,10 @@ def enter_data_using_post(pytestconfig, session):
+        str(pytestconfig.rootdir),
+        'tests',
+        'testmanifests',
+        'validexample0_archived.json5'
```
this was already fixed in the corresponding branch, but whatever

mharasic replied:
Came through dirty merge... only a single line

henderkes reviewed (commented):
I have zero knowledge about the frontend or javascript at all, so sorry for the short review.

mharasic committed:
Merge branch 'frontend/bugfixes/iss300' of https://github.com/Drakulix/knex into frontend/bugfixes/iss300

Conflicts:
	web/src/components/common/Filters.jsx
	web/src/components/views/CreateProject.jsx
	web/src/components/views/CreateProjectByPattern.jsx

mharasic committed:
minor changes

mharasic committed:
New Icons

mharasic committed:
proxy change

mharasic committed:
major refactoring

mharasic committed:
xxx

mharasic committed:
delete garbage

PSDavidSt committed:
Bugfix Delete Date

-changed the deleted date to null instead of an empty string
-> the delete button on date is now initial not visible

mharasic committed:
fixed share

mharasic committed:
Refactor & Fix

mharasic committed:
Merge branch 'frontend/bugfixes/iss300' of https://github.com/Drakulix/knex into frontend/bugfixes/iss300

mharasic committed:
minor change

mharasic committed:
Removed some Warnings

mharasic committed:
No Sharing for yourself

mharasic committed:
Prepared File Upload

mharasic committed:
File Upload

mharasic committed:
Removed alert

mharasic committed:
removed blinking

mharasic committed:
Minor change

mharasic committed:
minor changes

mharasic committed:
Bugfix BAAAD

mharasic committed:
BUGFIX Datepixker

mharasic committed:
Cosmetics

mharasic committed:
Deletion Bug of ChipInputs solved
Minor styling

mharasic committed:
Performance boost for Data Table

mharasic committed:
Filter Bug solved

mharasic committed:
MOAR Whitespace

mharasic committed:
Inkbar styling

mharasic committed:
Bug FIX

mharasic committed:
Merge branch 'master' into frontend/bugfixes/iss300

Rintel commented:
As I'm the author of the Pull Request, I can't approve it. Therefore one of you should please do it, @Drakulix is waiting for the deployment.

Phil-Schmidt committed:
minor fixups

- Removed typo in saved queries-tooltip
- resolved value display issue on CreateProjectByURL

pdler committed:
insert project from local json fixed

Phil-Schmidt committed:
JSON local file upload

- Json (single) file upload is now a thing, that exists.

Phil-Schmidt committed:
Merge pull request #317 from Drakulix/frontend/feature/localFileUpload

[FE] Local File Upload 2

Phil-Schmidt committed:
removed old local file upload

mharasic reviewed (dismissed):
Sounds working?

Phil-Schmidt committed:
More Snackbar usage

-some error messages in CreateProjectChoice were still using alerts as their output method. Now they will use the snackbar.

mharasic commented on web/src/components/views/CreateProjectFromFile.jsx:
```diff
@@ -0,0 +1,83 @@
+              project["description"] = data.description
+            }
+            if(data.status!=null){
+              switch(data.status.toLowerCase()){
```
status ist mitlerweile analog zum file definiert und ist nicht mehr zahlen basiert

mharasic commented on web/src/components/views/CreateProjectFromFile.jsx:
```diff
@@ -0,0 +1,83 @@
+            if(data.date_creation!=null){
+              var projectDate = new Date(data.date_creation)
+              this.setState({date : projectDate})
+              project["date_creation"] = data.date_creation
```
date_creation is okay.... but ...
date for the datepicker has to be defined as well

mharasic reviewed (changes requested):
See inline comments for 

https://github.com/Drakulix/knex/pull/310/commits/bca2ecb65d9a9ab36d816f57b169baba9dce5220

mharasic commented on web/src/components/views/CreateProjectFromFile.jsx:
```diff
@@ -0,0 +1,83 @@
+        super(props)
+        var data = this.props.json;
+        this.state = {
+          status : "2",
```
Status is now sting based DONE; IN_PROGRESS:...

mharasic committed:
fixed the bug!!!!

mharasic committed:
Color Bug fixed

pdler committed:
merge

Merge branch 'master' into frontend/bugfixes/iss300
